Error: The `form` element was not requested.
Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`. Note: If radio inputs are wrapped in their `label`, `br` is unnecessary.
diff --git a/output/Radio Group with Accessible Description/1/fea93e02-3daa-44b7-861d-56bd382558ed.html b/output/Radio Group with Accessible Description/1/fea93e02-3daa-44b7-861d-56bd382558ed.html
index c0f65e2..75529de 100644
--- a/output/Radio Group with Accessible Description/1/fea93e02-3daa-44b7-861d-56bd382558ed.html	
+++ b/output/Radio Group with Accessible Description/1/fea93e02-3daa-44b7-861d-56bd382558ed.html	
@@ -1,9 +1,13 @@
-<form>
-  <fieldset>
-    <legend>Choose a theme:</legend>
-    <label><input type="radio" name="theme" value="light"> Light</label><br>
-    <label><input type="radio" name="theme" value="dark"> Dark</label><br>
-    <label><input type="radio" name="theme" value="high-contrast"> High Contrast</label>
-    <p>You can change this later</p>
-  </fieldset>
-</form>
\ No newline at end of file
+<fieldset aria-describedby="theme-description">
+  <legend>Choose a theme:</legend>
+  <p id="theme-description">You can change this later</p>
+  <label for="light">
+    <input type="radio" name="theme" id="light" value="light"> Light
+  </label>
+  <label for="dark">
+    <input type="radio" name="theme" id="dark" value="dark"> Dark
+  </label>
+  <label for="high-contrast">
+    <input type="radio" name="theme" id="high-contrast" value="high-contrast"> High Contrast
+  </label>
+</fieldset>
\ No newline at end of file
